core/master/src/main/resources/config/commands/irc/core.conf

```diff
@@ -104,23 +104,56 @@ input all
 output source
 }
 
-change {
+addgroup {
+plugin org.drftpd.commands.usermanagement
+class GroupManagementHandler
+method doSITE_ADDGROUP
+perms =siteop
+help Add a new group
+help.specific ${command} <group>
+input all
+output source
+}
+
+changeuser {
 plugin org.drftpd.commands.usermanagement
 class UserManagementHandler
-method doSITE_CHANGE
+method doSITE_CHANGEUSER
 perms =gadmin =siteop
 help Change parameters for a user
 help.specific ${command} <username> <field> <value>
 help.specific Valid fields: group, created, ratio, idle_time, credits, tagline
-help.specific num_logins # [opt perIP], comment, group_ratio #min #max
-help.specific group_slots # [opt leech slots], wkly_allotment, max_sim #dn #up
+help.specific num_logins # [opt perIP], comment, wkly_allotment, max_sim #dn #up
+input all
+output source
+}
+
+changegroup {
+plugin org.drftpd.commands.usermanagement
+class GroupManagementHandler
+method doSITE_CHANGEGROUP
+perms =siteop
+help Change parameters for a group
+help.specific ${command} <groupname> <field> <value>
+help.specific Valid fields: created, slots, leechslots
+input all
+output source
+}
+
+changegroupadmin {
+plugin org.drftpd.commands.usermanagement
+class GroupManagementHandler
+method doSITE_CHANGEGROUPADMIN
+perms =siteop
+help Add/Remove users as group admin from a group
+help.specific ${command} <group> <user> [<user> ...]
 input all
 output source
 }
 
 chgrp {
```

Comment:
Same remark as FTP command

Reply:
Basically this is 'kind of' the same function as chgrp, but instead of becoming a member/leaving a group you become/get demoted admin of the group